README.md

```diff
@@ -0,0 +1,37 @@
+# Notebook data redirector
+
+This is an AWS application that redirects requests to files publicly hosted on Box.
+
+## Deployment
+
+To deploy the redirector, first install and configure the [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html).
+
+Next, run the following commands:
+
+```console
+$ sam build
+$ sam package --s3-bucket your-s3-bucket --output-template packaged.yaml
+$ sam deploy --template-file packaged.yaml --capabilities CAPABILITY_IAM --stack-name your-cloud-formation-stack --parameter-overrides SecretArn=your-secret-arn ManifestTableName=your-ddb-table-name
```

Comment:
specify which params need to be existing resources and which you can make up on the spot

Reply:
This has been addressed